dash/frontend/src/app/core/dialogues/show-json-data-more/share-event.component.ts

```diff
@@ -0,0 +1,43 @@
+import {Component, Inject, Input, OnInit} from '@angular/core';
```

Comment:
I believe Becky moved this file, so don't recreate it, you'll need to rebase onto main and make the change in the new version of the file

Reply:
It looks like you are still recreating this file